src/Components/Publishing/Byline/NewsByline.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react"
+import styled from "styled-components"
+import { getArticleFullHref, getAuthorByline, getDate } from "../Constants"
+import { Fonts } from "../Fonts"
+import { Share } from "./Share"
+
+interface NewsBylineProps {
+  article: any
+}
+
+export const NewsByline: React.SFC<NewsBylineProps> = props => {
+  const { article } = props
+  const { authors, title } = article
+  const url = getArticleFullHref(article.slug)
+
+  return (
+    <NewsBylineContainer>
+      <AuthorDateContainer>
+        <Poster>Posted by {getAuthorByline(authors)}</Poster>
+        <DateAndSource article={article} />
+      </AuthorDateContainer>
+      <Share url={url} title={title} />
+    </NewsBylineContainer>
+  )
+}
+
+const DateAndSource: React.SFC<NewsBylineProps> = props => {
+  const { article } = props
+  const { date, news_source } = article
+
+  return (
+    <DateSourceContainer>
+      {getDate(date, "verbose")}
+      {`, via  ${news_source.title}`}
```

Comment:
One thing to note is that not all articles will have a news_source so we want this to be a conditional